Nosenti/certificate-manager---Innocent
# Task4-KAN-111 Integrating APIs with Frontend

#### PR description
- This PR integrates the developed APIs on the frontend

#### Ticket / Jira Link
- [KAN-##](https://task-force-pro-dccs.atlassian.net/jira/software/projects/KAN/boards/1?assignee=712020%3Ab9bb79c7-4352-4d17-9af2-5c36278b1670&selectedIssue=KAN-)

#### Checklist:

#### How To Test Changes
- Inside the root repository, cd into backend and frontend
- Run the backend application
- Run the frontend application
- Use Frontend should be using the data from backend 

#### Screenshots
![image](https://github.com/user-attachments/assets/12a4f29a-948b-4205-8bd5-55c0d41ec1c3)
![image](https://github.com/user-attachments/assets/60123ed3-24a9-431a-aaef-e924dc596f2a)
![image](https://github.com/user-attachments/assets/9a9d5311-0da2-4ab2-86cf-4569f5abde66)
